lib/improver/cli/wet_bulb_temperature_integral.py

```diff
@@ -76,10 +51,11 @@ def process(wet_bulb_temperature):
         iris.cube.Cube:
             Processed Cube of wet bulb integrals.
     """
-    result = WetBulbTemperatureIntegral().process(
-        wet_bulb_temperature)
-    return result
+    from improver.psychrometric_calculations.psychrometric_calculations \
+        import WetBulbTemperatureIntegral
+    from improver.utilities.cli_utilities import make_cubes_non_lazy
```

Comment:
Eek! Ideally I'd like to scrap the whole of `improver.utilities.cli_utilities`.
I think the plugin itself should need to know whether or not it needs non-lazy data - could we move the touching of the data to there, please?  (Do we even need it?)

Reply:
(Aside from that I'm happy with this PR)